Minimatch/Minimatch.nuspec

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>$id$</id>
-    <version>$version$</version>
-    <title>$title$</title>
-    <authors>$author$</authors>
-    <owners>$author$</owners>
+    <id>Minimatch</id>
+    <version>1.1.0</version>
```

Comment:
Since nuget doesn't handle nuspecs with different names than the project, so I figured it would be better to just keep them up to date manually. Want me to revert it?